rasa_core/cli/arguments.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pkg_resources
+
+
+def add_config_arg(parser):
+    """Add an argument to the parser to request a policy configuration."""
+
+    parser.add_argument(
+            '-c', '--config',
+            type=str,
+            nargs="*",
+            default=[pkg_resources.resource_filename(__name__,
+                                                     "../default_config.yml")],
+            help="Policy specification yaml file.")
+    return parser
+
+
+def add_domain_arg(parser):
+    """Add an argument to the parser to request a the domain file."""
+
+    parser.add_argument(
+            '-d', '--domain',
+            type=str,
+            required=True,
+            help="domain specification yaml file")
+    return parser
```

Comment:
Can we make these functions return `None` to indicate that `parser` is mutable? We don't use the return value in any of these functions' calls